builder/builder.go

```diff
@@ -1737,13 +1741,47 @@ func (b *Builder) BuildImage(format string, template string) error {
 		format = b.State.Mix.Format
 	}
 
-	// If the user did not pass in a template, default to release-image-config.json
+	// swupd (client) called by itser will need a temporary directory to act as its stage dir.
+	wd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+
+	// If the user did not pass in a template, use the default
 	if template == "" {
-		template = "release-image-config.json"
+		template = imageTemplate
+
+		// If  the default image template is not present in the mix workspace,
+		// copy it from the default ister config directory and update the bundle list based on mix bundles
+		templateFile := filepath.Join(wd, template)
+		if _, err = os.Stat(templateFile); os.IsNotExist(err) {
+			fmt.Printf("Warning: Image template %s not found\n", templateFile)
```

Comment:
Please factor this into another function with a useful name like `copyTemplateFile(src, dest)` instead of all the code in this third indentation level.

Reply:
Done.